### DISCUSSION TITLE
Thank you so much for making this tool.

### DISCUSSION THREAD

**Discussion by GenericGeorge** [lacymorrow/crossover#300] [General]
I've used this every day for the past who knows! Many months, maybe a year :) Feels like it's part of my process now turn the PC on, and enable crosshair. Go bezerk on games! Love it. Couldn't go without this program anymore.

**Comment by lacymorrow** [comment]
Thanks for your kind words! I created this tool for myself, so seeing people use it is a really cool thing!

Cheers 🥂
